### PR TITLE
super FIx

### LINKED ISSUE #1: Plural minutes when no plural
# Big bad bug
As shown on the screenshot below the issue is quite obnoxious and makes the lib LITERALLY U N U S A B L E. Fix **as soon as possible** (asap in short (ais)).

khrsamy

Some suggestions:
```csharp
$"{(amount > 1 ? "minutes" : "minute")}"
```
or
```csharp
$"minute{(amount > 1 ? "s" : "")}"
```

### PR DESCRIPTION
**Pull request by Quahu**
this will save my family from hunger
Fixes #1

### CODE REVIEW

**Comment by Quahu** [comment]
and find me a gf most likely

File: src/Muscord/MusicRpcClient.cs [hunk 1]
```diff
@@ -145,4 +145,4 @@ public override void Connect()
             base.Connect();
         }
     }
-}
\ No newline at end of file
+}
```
Comment (Izumemori) [inline src/Muscord/MusicRpcClient.cs]: There's an extra newline at the end, could you remove that?

**Comment by Izumemori** [comment]
also add `Fixes #1` to your description

File: src/Muscord/MusicRpcClient.cs (hunk 1 above)
Comment (TurboLOL) [reply src/Muscord/MusicRpcClient.cs]: There is not Izu, what are you talking about?

Comment (Izumemori) [reply src/Muscord/MusicRpcClient.cs]: seems like I have to accept that, that's really not good style-wise, I will see how I fix that on my end

**Comment by Izumemori** [comment]
LGTM, Merging...

**Pull request merged**